src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -5657,6 +5657,16 @@ public override IOperation VisitInvocation(IInvocationOperation operation, int?
                                            operation.Type, IsImplicit(operation));
         }
 
+        public override IOperation? VisitFunctionPointerInvocation(IFunctionPointerInvocationOperation operation, int? argument)
+        {
+            EvalStackFrame frame = PushStackFrame();
+            IOperation? pointer = operation.Target;
+            (IOperation? visitedPointer, ImmutableArray<IArgumentOperation> visitedArguments) = VisitInstanceWithArguments(pointer, operation.Arguments);
```

Comment:
The target isn't an `instance`, and we shouldn't use this helper here. I'm working on a separate interpolated strings CFG change that will touch that code path, and it shouldn't affect function pointers.

Reply:
I'm not shure what the meaning of Push and Pop is...

Reply:
For now I created a VisitFunctionPointerWithArguments function - but I'm not sure if it is nice or needed.